frameworks/Nim/nawak/setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+export PATH="$IROOT/nim/bin:$IROOT/mongrel2/bin:$PATH"
+
+sed -i 's|host=.* port=5432|host='"${DBHOST}"' port=5432|g' model_postgre.nim
+
+echo "** Compiling app"
+nim c --threads:on -d:release -d:postgre_model -l:-Wl,-rpath,$IROOT/zeromq-4.0.3/lib -o:nawak_postgre app.nim
+
+echo "** Starting mongrel2"
+cd conf
+mkdir -p run logs tmp
+m2sh load -config mongrel2.conf
+m2sh start -name test &
```

Comment:
I pulled this and tested it manually, it seems to be working after a few tweaks, but I'm not sure why Travis thinks that it isn't.